packages/web3-core/CHANGELOG.md

```diff
@@ -40,3 +40,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 -   If the response error was `execution reverted`, raise `ContractExecutionError` and pass the response error to it in order to be set as `innerError` (this innerError will be decoded at web3-eth-contract if its ABI was provided according to EIP-838). (#5434)
+-   `registerPlugin` method to `Web3Context` (#5393)
```

Comment:
Could all the changes to `CHANGELOG.md` files be duplicated on the one at the root?

Reply:
Updated via [this](https://github.com/web3/web3.js/pull/5393/commits/bba74e69e1985e06d4a971701b8e5598a8636522) commit